web/src/frontend/src/nav-soknad/redux/valideringReducer.ts

```diff
@@ -51,6 +51,26 @@ const unregisterFaktumValidering = (
 	];
 };
 
+const setFaktumValideringsfeil = (
+	feil: Valideringsfeil[],
+	faktumKey: string,
+	faktumValideringfeil: Valideringsfeil[]
+) => {
+	if (faktumValideringfeil.length === 0) {
+		return feil.filter(v => v.faktumKey !== faktumKey);
+	}
+	const idx = feil.findIndex(v => v.faktumKey === faktumKey);
+	if (idx === -1) {
+		return feil.concat(faktumValideringfeil);
+	}
+	// For å ivareta samme rekkefølge på feilmeldingene i oppsummeringen
+	return [
```

Comment:
Vurder å bruke splice i stedet for slice. I stedet for dette:
```
   return [
		...valideringsregler.slice(0, idx),
		...valideringsregler.slice(idx + 1)
	]
```

kan man bruke dette:

```
return valideringsregler.splice(idx, 1); // remove 1 item at 2-index position 
```